src/main/java/dev/tourmi/svmm/utils/MinecraftUtils.java

```diff
@@ -59,19 +59,21 @@ public static boolean isBlockInList(BlockState bs, List<? extends String> list)
     }
 
     public static void mineBlock(Level level, Player player, BlockPos blockPos, ItemStack heldItem) {
-        if (level.isClientSide) return;
+        if (level.isClientSide()) return;
 
         var blockState = level.getBlockState(blockPos);
         heldItem.mineBlock(level, blockState, blockPos, player);
         blockState.getBlock().popExperience((ServerLevel) level, blockPos, EnchantUtils.getExp(level, heldItem, blockPos, blockState));
         blockState.getBlock().playerDestroy(level, player, blockPos, blockState, level.getBlockEntity(blockPos), heldItem);
         level.removeBlock(blockPos, false);
-        if (SVMMConfig.TELEPORT_ITEMS_TO_PLAYER.get()) {
-            BLOCKS_MINED.put(blockPos, player);
-            level.getServer().tell(new TickTask(level.getServer().getTickCount() + 10, () -> {
-                BLOCKS_MINED.remove(blockPos);
-            }));
-        }
+
+		if (SVMMConfig.TELEPORT_ITEMS_TO_PLAYER.get()) {
+			BLOCKS_MINED.put(blockPos, player);
+
+			((ServerLevel) level).getServer().execute(() -> {
+				BLOCKS_MINED.remove(blockPos);
+			});
+		}
```

Comment:
I noticed that `tell` was renamed to `schedule` in 1.21.3, however, after testing, the 10 ticks delay appear to not have been needed after all, so a call to `execute` works, making this change compatible with both `1.21` and `1.21.3`!